packages/x-charts/src/internals/stackSeries.ts

```diff
@@ -1,91 +1,125 @@
 import {
-  stack as d3Stack,
+  stackOrderAppearance as d3StackOrderAppearance,
+  stackOrderAscending as d3StackOrderAscending,
+  stackOrderDescending as d3StackOrderDescending,
+  stackOrderInsideOut as d3StackOrderInsideOut,
   stackOrderNone as d3StackOrderNone,
+  stackOrderReverse as d3StackOrderReverse,
+  stackOffsetExpand as d3StackOffsetExpand,
+  stackOffsetDiverging as d3StackOffsetDiverging,
   stackOffsetNone as d3StackOffsetNone,
+  stackOffsetSilhouette as d3StackOffsetSilhouette,
+  stackOffsetWiggle as d3StackOffsetWiggle,
+  Series,
 } from 'd3-shape';
-import { BarSeriesType, LineSeriesType, StackableSeriesType } from '../models/seriesType';
+import { BarSeriesType, LineSeriesType } from '../models/seriesType';
 
 type StackableSeries = { [id: string]: BarSeriesType } | { [id: string]: LineSeriesType };
 
 type FormatterParams = { series: StackableSeries; seriesOrder: string[] };
 
+export type StackingGroupsType = {
+  ids: string[];
+  stackingOrder: (series: Series<any, any>) => number[];
+  stackingOffset: (series: Series<any, any>, order: Iterable<number>) => void;
+}[];
+
+export const StackOrder: {
+  [key in 'appearance' | 'ascending' | 'descending' | 'insideOut' | 'none' | 'reverse']: (
```

Comment:
Maybe it would make sense to have this as a `StackOrderType`? 🤔 
It could then be used in the `StackOrderDemo` instead of duplication. 🤔